lib/countly.js

```diff
@@ -491,8 +495,6 @@
                 //empty event queue
                 if (eventQueue.length > 0) {
                     toRequestQueue({ events: JSON.stringify(eventQueue) });
-                    eventQueue = [];
-                    store("cly_event", eventQueue);
```

Comment:
Sorry, if there was a misunderstanding, but we need to empty queue on `change_id`, what I meant was, you dont use it in tests, so no need to do anything test specific here :)